features/options/profiles/version0.feature

```diff
@@ -71,7 +72,7 @@ end
 
 function turn_function (angle)
   print('turn_function ' .. angle)
-  return angle == 0 and 0 or 42
+  return angle == 0 and 0 or 17
```

Comment:
just a dummy number to have expectations 15+1.7 not so close to 20 as 15+4.2 for 42.
Basically here i did not get why a-e path is 20s, but a-b and a-d is 15+turn_penalty, 
i would expect 20+turn_penalty